### DISCUSSION TITLE
Is there a plan to add a graph option like this one?

### DISCUSSION THREAD

**Discussion by gusandrianos** [patrykandpatrick/vico#413] [Ideas]
Hello!

We're using Vico for an investments feature and I was wondering if there is a plan to add a graph like this:
![image](https://github.com/patrykandpatrick/vico/assets/17255848/d15bb371-95fb-484d-a470-14610f604147)

Is there maybe a way to achieve this result with the current implementation of Vico?

Thank you!

**Comment by patrickmichalik** [comment]
Hello! I’m afraid there’s no such feature at the moment, but we’re currently looking into adding it, so it should be available shortly. We’ll keep you updated. (You could, as such, create this kind of chart with Vico, but you’d have to override large portions of the `LineChart` class.)

**Reply by gusandrianos** [reply]
Thanks a lot, Patrick!

**Comment by patrickmichalik** [comment]
Hello! [Vico 2.0.0 Alpha 1](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.1) introduces this functionality. Let me know if you have any questions.

**Reply by gusandrianos** [reply]
That's awesome! I'll try to find some time to test it, thank you :)

**Reply by patrickmichalik** [reply]
Sounds good. You’re welcome!

**Comment by Gowsky** [comment]
Hello, [Vico 2.0.0](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0) is now available.
